packages/geo/src/lib/filter/ogc-filter-toggle-button/ogc-filter-toggle-button.component.html

```diff
@@ -0,0 +1,21 @@
+<ng-container *ngFor="let bundle of pushButtonBundle">
+    <ng-container *ngIf="bundleIsVertical(bundle)">
+        <mat-button-toggle-group vertical multiple="true">
+            <mat-button-toggle [ngStyle]="setColor(ogcPushButton)" [checked]="ogcPushButton.enabled"
+                (change)="applyFilters(ogcPushButton)" tooltip-position="below" matTooltipShowDelay="500"
+                [matTooltip]="getToolTip(ogcPushButton)" *ngFor="let ogcPushButton of bundle.ogcPushButtons"
+                [value]="ogcPushButton">{{ogcPushButton.title}}
+            </mat-button-toggle>
+        </mat-button-toggle-group>
+    </ng-container>
+    <!-- This duplicate is there because I'm not able to bind a value to vertical= "false" or "true" -->
```

Comment:
Did you try using a if/else with ng-template?

Reply:
Resolved by binding a property on vertical attribute.